go.mod

```diff
@@ -13,8 +13,8 @@ require (
 	go.opencensus.io v0.24.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.24.0
-	go.viam.com/api v0.1.179-0.20230814212331-d05fed6b5dfc
-	go.viam.com/rdk v0.7.3
+	go.viam.com/api v0.1.186
```

Comment:
[q] did you do a a `go get -u` on api? Nominally we want it linked to the latest instead of stable.

Reply:
I did not - this was auto updated when I bumped RDK.